fix(docs): removes references to `getInitialTestAccountsWallets`

sklppy88 opened the pull request:
This removes this deprecated reference in the docs/ directory, for new releases

Leaving the `2.0.2` versioned docs unchanged as it exists in that version

This has been tested with the latest nightlies on m1 mac & yarn 4.9 / node 24

critesjosh commented on docs/docs/developers/tutorials/contract_tutorials/token_contract.md:
```diff
@@ -237,38 +237,41 @@ Create `index.ts`. We will connect to our running sandbox and its PXE, then depl
-import { BobTokenContract } from './artifacts/BobToken.js';
-import { createPXEClient, waitForPXE, Wallet } from '@aztec/aztec.js';
-import { getInitialTestAccountsWallets } from '@aztec/accounts/testing';
+import { BobTokenContract } from './BobToken.js';
```
```suggestion
import { BobTokenContract } from './artifacts/BobToken.js';
```
following the instructions above puts it in `artifacts/`

sklppy88 replied:
Ah I was on mainframe and my setup was a bit custom and this slipped in, thanks for they 👀

critesjosh commented on docs/docs/developers/docs/tutorials/contract_tutorials/token_contract.md:
```diff
@@ -237,38 +237,41 @@ Create `index.ts`. We will connect to our running sandbox and its PXE, then depl
+import { BobTokenContract } from './BobToken.js';
+import { createAztecNodeClient } from '@aztec/aztec.js';
+import { getInitialTestAccountsData } from '@aztec/accounts/testing';
+import { TestWallet } from '@aztec/test-wallet/server';
```
I think this package needs to be imported at the top of the tutorial.

sklppy88 replied:
Thanks for the catch !

critesjosh commented on docs/docs/developers/docs/tutorials/contract_tutorials/token_contract.md:
````diff
@@ -237,38 +237,41 @@ Create `index.ts`. We will connect to our running sandbox and its PXE, then depl
 Then we will use the `giggleWallet` to deploy our contract, mint 100 BOB to Alice, then transfer 10 of those to Bob's Clinic publicly... for now. Let's go:
 
 ```typescript
````
Does this script work for you? I'm getting

```bash
josh in ~/Documents/test/token-tutorial on master ● ● λ npx tsx index.ts
Error: /home/josh/Documents/test/token-tutorial/node_modules/@aztec/native/build/nodejs_module.node: file too short
    at Object..node (node:internal/modules/cjs/loader:1921:18)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Module.require (node:internal/modules/cjs/loader:1487:12)
    at require (node:internal/modules/helpers:135:16)
    at bindings (/home/josh/Documents/test/token-tutorial/node_modules/bindings/bindings.js:112:48)
    at file:///home/josh/Documents/test/token-tutorial/node_modules/@aztec/native/dest/native_module.js:2:22
    at ModuleJob.run (node:internal/modules/esm/module_job:274:25) {
  code: 'ERR_DLOPEN_FAILED'
}
```

sklppy88 replied:
Ah I was on mainframe, but I think you're on mac, which explains another comment as well. Have followed the tutorial with my mac and have added the requirements ! Note I'm using a tmpstore on purpose for simplicity here. 

Also it needs have the nodelinker set as otherwise I was getting pnp from yarn 4.9, could you confirm this is the case on your end as well ?

critesjosh reviewed (changes requested):
At the beginning of the tutorial I think we should tell people to run `yarn init --type module` instead of just 'yarn init'

sklppy88 commented:
> At the beginning of the tutorial I think we should tell people to run `yarn init --type module` instead of just 'yarn init'

Hmm, I'm getting `Unknown Syntax Error: Unsupported option name ("--type").` when trying that command and I don't see it on their docs. I've addressed the other comments though, thanks for the review !

critesjosh commented on docs/docs/developers/tutorials/contract_tutorials/token_contract.md:
```diff
@@ -238,37 +240,45 @@ Then we will use the `giggleWallet` to deploy our contract, mint 100 BOB to Alic
 import { BobTokenContract } from './artifacts/BobToken.js';
-import { createPXEClient, waitForPXE, Wallet } from '@aztec/aztec.js';
-import { getInitialTestAccountsWallets } from '@aztec/accounts/testing';
+import { createAztecNodeClient } from '@aztec/aztec.js';
```
```suggestion
import { createAztecNodeClient, Wallet } from '@aztec/aztec.js';
```
I had to import Wallet bc its used in the `getBalances` fn below

critesjosh commented on docs/docs/developers/tutorials/contract_tutorials/token_contract.md:
```diff
@@ -238,37 +240,45 @@ Then we will use the `giggleWallet` to deploy our contract, mint 100 BOB to Alic
-        .mint_public(aliceWallet.address, 100n)
-        .send({ from: giggleWallet.address })
+    await bobToken.methods
+        .mint_public(aliceAccount.getAddress(), 100n)
```
The `index.ts` at the end of the tutorial needs to be updated with these changes as well.

sklppy88 committed:
fix(docs): removes references to `getInitialTestAccountsWallets`

This removes this deprecated reference in the docs/ directory, for new releases

Leaving the `2.0.2` versioned docs unchanged as it exists in that version

This has been tested with the latest nightlies on m1 mac & yarn 4.9 / node 24
